src/promptflow-evals/pyproject.toml

```diff
@@ -0,0 +1,96 @@
+[build-system]
+requires = ["poetry-core>=1.5.0"]
+build-backend = "poetry.core.masonry.api"
+
+# poetry
+[tool.poetry]
+name = "promptflow-evals"
+version = "0.1.0b1"
+description = "Prompt flow evals"
+license = "MIT"
+authors = [
+  "Microsoft Corporation <aml-pt-eng@microsoft.com>"
```

Comment:
do you guys have a team email ?

Reply:
@singankit: I think we can use azuremlsdk@microsoft.com for now. What do you think?

Reply:
Changed it to azuremlsdk@microsoft.com